RestaurantManagementSystem/RestaurantManagementSystem/src/web/Controllers/RestaurantController.cs

```diff
@@ -10,24 +10,23 @@ namespace RestaurantManagementSystem.src.web.Controllers
     public class RestaurantController: ControllerBase
     {
         private readonly IRegisterRequestCommandHandler _requesthandler;
-        private readonly IEmailConfirmationHandler  _emailhandler;
+        private readonly IEmailConfirmationHandler  _emailconfirmationhandler;
         public RestaurantController (IRegisterRequestCommandHandler requesthandler, IEmailConfirmationHandler emailConfirmation)
         {
            _requesthandler = requesthandler;
-           _emailhandler = emailConfirmation;
+           _emailconfirmationhandler = emailConfirmation;
```

Comment:
this is not an email confirmation handler, this is a VerifyRequestCommandHandler